packages/did-ethr-resolver/src/implementations/operator.ts

```diff
@@ -50,16 +54,25 @@ export class Operator extends Resolver implements IOperator {
   protected readonly _owner: IdentityOwner;
 
   /**
- * @param { IdentityOwner } owner - entity which controls document updatable by this operator
- */
-  constructor(owner: IdentityOwner, settings: RegistrySettings) {
-    super(owner.provider as providers.Provider, settings);
+  * @param { string | providers.Web3Provider } privKeyOrprovider - privateKey or Web3provider instance 
+  *                                                                of the entity which controls document
+  * @param settings - Settings to connect to Ethr registry
+  * @param { string } publicKey - publicKey of the entity which controls document
+  * @param { string } providerUrl - Connection link to the blockchain
+  */
+  constructor(
+    privKeyOrprovider: string | providers.Web3Provider,
+    settings: RegistrySettings,
+    publicKey: string,
+    providerUrl?: string
```

Comment:
I know this might feel like 2 steps forward 1 step back, but I think it would be better to go back using an `IdentityOwner` in the constructor as it was before 😄  . `IdentityOwner` nicely encapsulated the signer, publicKey and provider 👍